### DISCUSSION TITLE
Fundamentos c#  Value Types e Reference

### DISCUSSION THREAD

**Discussion by osvaldomachado** [balta-io/2801#25] [Perguntas e Respostas] [answered]
No slide (0:22) Heap e Stack

Informa que Heap armazena os dados e Stack as referências...

Mas na explicação da para entender que de fato é o contrário.

Podem me confirmar por favor?

Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @osvaldomachado como vai?
Você está correto, me diz por favor qual aula está assim que vou corrigir!

**Reply by osvaldomachado** [reply]
Oi.
A aula está em 
1 Fundamentos c# 
   > Value Types e Reference
       >  Slide (tempo 0:22) Heap e Stack

**Reply by andrebaltieri** [reply]
Muito obrigado @osvaldomachado 💜
